frontend/server/src/Exceptions/TokenValidateException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace OmegaUp\Exceptions;
+
+class TokenValidateException extends \Exception {
+    /**
+     * @readonly
+     * @var array<string, string>
+     */
+    public $claims;
+
+    /**
+     * @param string $message
+     * @param array<string, string> $claims
+     */
+    public function __construct($message = 'tokenDecodeCorrupt', $claims = []) {
+        parent::__construct($message, 400);
```

Comment:
```suggestion
        parent::__construct($message);
```

para `\Exception`, el segundo parámetro no tiene mucho sentido.

Reply:
Se va.